### DISCUSSION TITLE
We lost the previous activity and observability

### DISCUSSION THREAD

**Discussion by falsandtru** [community/community#37403] [Other Feature Feedback, Questions, & Ideas]
No way to restore the previous activity. We can no longer monitor the commits of the bot of GitHub Actions. Feed is just annoying.

**Comment by mikekavouras** [comment]
Hey @falsandtru. I'm curious to understand what activity you're referring to. Would you mind explaining what you mean by "previous activity"?

* Where did you see this activity?
* Is the activity from a specific repository?
* Is the repository part of an organization?

**Reply by falsandtru** [reply]
- Where did you see this activity?
  - Top page
- Is the activity from a specific repository?
  - My repositories
- Is the repository part of an organization?
  - Not sure because I'm not using organizations
